translations/pt-PT.yml

```diff
@@ -125,4 +125,4 @@ pt-PT:
       - question: "Qual é o símbolo para a moeda Bitcoin Cash?"
         answer:
           - |
-            A moeda Bitcoin Cash é representada por uma quantidade de símbolos de registo diferentes, dependendo do serviço ou carteira. BCH/BCC são os símbolos mais populares com XBC a ser utilizado para corresponder a norma internacional para códigos de moedas (ISO 4217).
+            A moeda Bitcoin Cash é representada por uma quantidade de símbolos de registo diferentes, dependendo do serviço ou carteira. BCH/BCC são os símbolos mais populares com XBC a ser utilizado para corresponder à norma internacional para códigos de moedas (ISO 4217).
```

Comment:
NIT: Might be worth updating this to only mention BCH like the english translation.